inc_internal/message.h

```diff
@@ -55,6 +55,9 @@ typedef struct {
     uint8_t *value;
 } hdr_t;
 
+#define var_header(id, var) header(id, sizeof(var), &(var))
```

Comment:
why not to use these on the connect side too?

Reply:
eventually they'll make it there